docs(api): adjust pump toggle values (#793)
flybywiresim/docs: docs/fbw-a32nx/a32nx-api/a32nx-flightdeck-api.md

Flight Deck: [Fuel Panel](../../pilots-corner/a32nx-briefing/flight-deck/ovhd/fuel.md)

!!! note "The below table shows the API for PUMP 1. Replace with appropriate value for the other pumps."

>     L1=2, L2=5. C1=9, C2=10, R1=3, R2=6

Reviewer: ```suggestion
    L1=2, L2=5, C1=9, C2=10, R1=3, R2=6
```

    !!! warning ""

>         Please note that FUELSYSTEM_PUMP_TOGGLE 1 and 4 for the center tank pump switches don't work anymore because these pumps do not exist in the NEO, which was corrected in a recent update.

Reviewer: ```suggestion
        Please note that FUELSYSTEM_PUMP_TOGGLE 1 and 4 for the center tank pump switches got replaced with FUELSYSTEM_VALVE_TOGGLE 9 and 10, due to the NEO having jet pumps instead of conventional pumps, which was corrected in a recent update.
```

        
        The new vars or events for these buttons are FUELSYSTEM_VALVE_TOGGLE 9 and 10.

| Function    | API Usage                 | Values        | Read/Write | Type       | Remark |
|:------------|:--------------------------|:--------------|:-----------|:-----------|:-------|

> | Fuel Pump 1 | FUELSYSTEM_PUMP_TOGGLE    | 2,3,5,6,9,10  | -          | MSFS EVENT |        |

Reviewer: ```suggestion
| Fuel Pump 1 | FUELSYSTEM_PUMP_TOGGLE    | 2,3,5,6  | -          | MSFS EVENT |  Fuel pumps only available in the wing tanks |
| Fuel Jet Pump 1 | FUELSYSTEM_VALVE_TOGGLE    | 9,10  | -          | MSFS EVENT | Used for center tank fuel jet pump valves |
```

Reviewer: New line probably best under the VAR lines for the pumps.

Author: Just had to test this locally since on github it doesn't look very pretty. Thanks I missed the actual API changes reading the issue for some reason.

Flight Deck: [Fuel Panel](../../pilots-corner/a32nx-briefing/flight-deck/ovhd/fuel.md)

!!! note "The below table shows the API for PUMP 1. Replace with appropriate value for the other pumps."
    L1=2, L2=5, C1=9, C2=10, R1=3, R2=6

    !!! warning ""
        Please note that FUELSYSTEM_PUMP_TOGGLE 1 and 4 for the center tank pump switches got replaced with FUELSYSTEM_VALVE_TOGGLE 9 and 10, due to the NEO having jet pumps instead of conventional pumps, which was corrected in a recent update.

>         
>         The new vars or events for these buttons are FUELSYSTEM_VALVE_TOGGLE 9 and 10.

Reviewer: ```suggestion
```

Flight Deck: [Fuel Panel](../../pilots-corner/a32nx-briefing/flight-deck/ovhd/fuel.md)

!!! note "The below table shows the API for PUMP 1. Replace with appropriate value for the other pumps."
    L1=2, L2=5, C1=9, C2=10, R1=3, R2=6

    !!! warning ""
        Please note that FUELSYSTEM_PUMP_TOGGLE 1 and 4 for the center tank pump switches got replaced with FUELSYSTEM_VALVE_TOGGLE 9 and 10, due to the NEO having jet pumps instead of conventional pumps, which was corrected in a recent update.
        

>         The new vars or events for these buttons are FUELSYSTEM_VALVE_TOGGLE 9 and 10.

Reviewer: ```suggestion
```
